src/gstreamer/gstkvssink.cpp

```diff
@@ -1002,6 +1042,15 @@ gst_kvs_sink_get_property(GObject *object, guint prop_id, GValue *value,
         case PROP_STOP_STREAM_TIMEOUT:
             g_value_set_uint (value, kvssink->stop_stream_timeout);
             break;
+
+        case PROP_SERVICE_CONNECTION_TIMEOUT:
+            g_value_set_uint (value, kvssink->stop_stream_timeout);
+            break;
+
+        case PROP_SERVICE_COMPLETION_TIMEOUT:
+            g_value_set_uint (value, kvssink->stop_stream_timeout);
+            break;
+
```

Comment:
copy/paste TYPO here these are all being set to stop_stream_time, need to be set correctly.

Reply:
Oops. Fixed.